### PR TITLE
run grinder in checked mode on the bot

### PR DESCRIPTION
**Pull request by devoncarew**
Related to #222, run grinder in checked mode on the bots (and fix one checked mode error).

### CODE REVIEW

File: lib/src/cli.dart [hunk 1]
```diff
@@ -35,12 +35,13 @@ cli(
     @Rest(help: getTaskHelp, allowed: allowedTasks)
     List<String> tasks,
     {
-     @Flag(help: 'Print the version of grinder.')
+     @Flag(help: 'Print the version of grinder.', defaultsTo: false)
      bool version,
```
Comment (seaneagan) [inline lib/src/cli.dart]: Nit: You can just do `bool version: false`.  I like to define as much of the interface using dart-native features as possible.

File: .travis.yml [hunk 2]
```diff
@@ -1,3 +1,3 @@
 language: dart
-script: pub run grinder:grinder
+script: dart -c tool/grind.dart buildbot
```
Comment (seaneagan) [inline .travis.yml]: Nit: Can we remove `buildbot` to dogfood the default task handling?

File: lib/src/cli.dart (hunk 1 above)
Comment (devoncarew) [inline lib/src/cli.dart]: sgtm. I like this better as well; I got a bit confused by the optional params and param annotations -

File: .travis.yml (hunk 2 above)
Comment (devoncarew) [inline .travis.yml]: done

**Comment by devoncarew** [comment]
updated -

File: .project [hunk 3]
```diff
@@ -10,9 +10,15 @@
 			<arguments>
```
Comment (seaneagan) [inline .project]: Should this file be .gitignored?

Comment (seaneagan) [inline .project]: I guess this file could probably be .gitignore'd (I thought it was already in github's default .gitignore for Dart), not that it hurts anything having it.

Comment (devoncarew) [inline .project]: Hmm, in any case I didn't mean to have it in this PR -

**Comment by seaneagan** [comment]
lgtm

**Pull request merged**